pyccel/parser/scope.py

```diff
@@ -87,6 +92,8 @@ def __init__(self, *, name=None, decorators = (), is_loop = False,
         self._dummy_counter = 0
 
         self._locals['decorators'].update(decorators)
+        if symbolic_aliases:
+            self._locals['symbolic_alias'].update(symbolic_aliases)
```

Comment:
For consistency, maybe you could change the key name `symbolic_alias` to `symbolic_aliases` (plural).

Reply:
Fixed 339892ae6